Katello/katello
# Fixes #27450: Check permissions for React pages

To test this PR:

1.  Create a User with no roles (no permissions)
2.  Log in as (or impersonate) that user.  
3.  Navigate manually (by typing in the URL bar) to `/subscriptions` and `/redhat_repositories`. 

You should now see the permission denied message for those two pages.

[Edit: Please also test "Flow 1" (see below)]
